src/app/page.tsx

```diff
@@ -11,3 +32,11 @@ export default function Home() {
     </div>
   );
 }
+
+export default function Home() {
+  return (
+    <Suspense fallback={<div>Loading...</div>}>
```

Comment:
ideally we should have a nice loading component but that is nice to show to user when the site is loading but that is nice to have feature so we can deal with it later on